secure/README.md

```diff
@@ -0,0 +1,34 @@
+# Securing Dataproc
+
+This initialization action installs the MIT distribution of the Kerberos packages, and configure Yarn, HDFS, Hive, Spark to integrate with Kerberose, which enables in-transit data encryption.
+It will also encrypt HTTP traffic (include Web UIs and shuffle) with SSL.
+
+## Using this initialization action
+You can use this initialization action to create a secured Dataproc cluster:
+  Using the `gcloud` command to create a new cluster with this initialization action.
+
+```bash
+gcloud dataproc clusters create <CLUSTER_NAME> \
+    --scopes cloud-platform \
+    --initialization-actions gs://dataproc-initialization-actions/secure/secure.sh \
+    --metadata "kms-key-uri=projects/<PROJECT_ID>/locations/global/keyRings/my-key-rings/cryptoKeys/my-key" \
+    --metadata "keystore-uri=gs://<SECRET-BUCKET>/keystore.jks" \
+    --metadata "truststore-uri=gs://<SECRET-BUCKET>/truststore.jks" \
+    --metadata "db-password-uri=gs://<SECRET-BUCKET>/dp-password.encrypted" \
+    --metadata "root-password-uri=gs://<SECRET-BUCKET>/root-password.encrypted" \
+    --metadata "keystore-password-uri=gs://<SECRET-BUCKET>/keystore-password.encrypted"
+```
+
+All the metadata key-value pairs are required to properly secure the cluster.
+
+1. Use **keystore-uri** to specify the GCS location of the keystore file which contains the SSL certificate. It has to be in the Java KeyStore (JKS) format and when copied to VMs, renamed (if necessary) to keystore.jks. The SSL certificate should be a wildcard certificate which applies to every node in the cluster.
```

Comment:
Could you add some background information about keystore and truststore? Also add help for how to generate (or get) a keystore and truststore.

Reply:
I added example commands that generate keystore/truststore.